mu4e-thread-folding.el

```diff
@@ -180,11 +181,11 @@ This uses the mu4e private API and this might break in future releases."
                                           (+ child-prefix-end (line-beginning-position))))
 
                    ;; Overlay for child (whole line)
-                   (child-overlay  (make-overlay
-                                    (+ 0 (line-beginning-position))
-                                    (+ 1 (line-end-position)))))
                
-               ;; We mark the root thread if and only if there'a child               
+               ;; We mark the root thread if and only if there's child
+                   (child-overlay (make-overlay
```

Comment:
Not sure I understand why you moved this part

Reply:
Double space removed:

```diff
-child-overlay  (make-overlay
+child-overlay (make-overlay
```

Reply:
Wait, something strange has happened there.  I will fix and re-push.

Reply:
Sorry about that.  I'd done some initial reformatting on an older revision of the code, and I obviously messed up when rebasing it.